scapy/layers/l2.py

```diff
@@ -474,7 +474,7 @@ class GRE(Packet):
                    ConditionalField(XShortField("chksum", None), lambda pkt:pkt.chksum_present == 1 or pkt.routing_present == 1),  # noqa: E501
                    ConditionalField(XShortField("offset", None), lambda pkt:pkt.chksum_present == 1 or pkt.routing_present == 1),  # noqa: E501
                    ConditionalField(XIntField("key", None), lambda pkt:pkt.key_present == 1),  # noqa: E501
-                   ConditionalField(XIntField("seqence_number", None), lambda pkt:pkt.seqnum_present == 1),  # noqa: E501
```

Comment:
Please use the `deprecated_fields` functionality as shown here https://github.com/secdev/scapy/blob/master/scapy/layers/dot11.py#L244. 

Also, could you rename the fields in the unit tests https://github.com/secdev/scapy/blob/master/test/pptp.uts#L44 ?